src/storage/mod.rs

```diff
@@ -386,6 +416,10 @@ pub struct Options {
     pub skip_constraint_check: bool,
     pub key_only: bool,
     pub reverse_scan: bool,
+    pub is_first_lock: bool,
+    pub should_not_exist: bool,
```

Comment:
This is only used in pessimistic lock requests, can we remove it and make it an argument?